### DISCUSSION TITLE
Help: Another Custom Snippet Not Working Question

### DISCUSSION THREAD

**Discussion by nickali** [LazyVim/LazyVim#3678] [General]
I've started with a clean install of lazyvim.

Looking through the other discussions, this is what I've tried:

1) Created ~/.config/nvim/lua/config/snippets.lua. It's contents:

```
return {
        "garymjr/nvim-snippets",
        dependencies = {
          "rafamadriz/friendly-snippets",
        },
        -- dev = true,
        opts = {
          global_snippets = { "all", "global" },
          search_paths = { vim.fn.stdpath("config") .. "/snippets" },
        }
}
```

2) Created a ~/.config/nvim/lua/config/snippets/all.lua for testing. It's contents:

```
return {
  s("trig", t("loaded!!")),
}
```

Then I opened up a random markdown file and typed 'trig' and got a pop with 'transitioning - Text" as the only option.

3) Created ~/.config/nvim/lua/config/snippets/all/all.json with the contents.

```
{
  "trig": {
    "prefix": "trig",
    "description": "trig snippet",
    "body": [
      "trigger loaded"
    ]
  },
  "Just a test": {
    "prefix": "mytest",
    "description": "Just a test",
    "body": [
      "This is just a test"
    ]
  }
}
```

'trig' still doesn't trigger snippets. 

What am I doing wrong?

**Comment by dpetka2001** [comment]
You must put your customizations in `~/.config/nvim/lua/plugins/` directory not `/lua/config/` directory. 

The 1st step is not needed at all since they are all default values. 
The 2nd step is syntax for Luasnip, so it won't work with native snippets. 
The 3rd step is actually the only one needed. The first 2 steps are not needed in what you're trying to do. For the 3rd step you should put your snippets in `~/.config/nvim/snippets/` directory.

**Reply by nickali** [reply]
That worked. Thanks.

**Reply by a-torgovitsky** [reply]
Is this still the way to do it? I tried two approaches:
1. Create `~/.config/nvim/snippets/all.json` with the `all.json` file that @nickali put above.
2. Create `~/.config/nvim/snippets/all/all.json`

But either way I don't see either `trig` or `mytest` showing up as snippets.
I haven't changed any of the defaults for `nvim-snippets`.

**Reply by nickali** [reply]
@a-torgovitsky `~/.config/nvim/snippets/all/all.json` is what worked for me. 

I use AstroNvim and used their community repository to [set up nvim-snippets](https://github.com/AstroNvim/astrocommunity/tree/main/lua/astrocommunity/snippet/nvim-snippets). Maybe the init.lua could help you. I haven't changed any of those default settings.

**Reply by a-torgovitsky** [reply]
Thanks. I figured out the issue was that I had luasnips enabled as a lazyvim extra. Disabling it solved the problem.
